CHANGELOG.md

```diff
@@ -37,6 +32,22 @@ Versioning](https://semver.org/spec/v2.0.0.html).
 -
 
 
+## [0.1.51] - 2022-05-10
+
+### Enhancements
+
+- Added `maunalRules` to `TailoredProfile` CRD, user can choose to add the rule
+  there so that those rules will show Manual as results and remediations will not be 
+  created.
+
+### Fixes
+
+- Fix OpenScap scanner container crashloop caused by script mount permission issue
+  on a security envronment where DAC_OVERRIDE capability is dropped. This PR changes
+  script mount permission to give excute permission to all users.
```

Comment:
Bah - I missed one! (s/excute/execute)

Reply:
let me fix it

Reply:
I should have Grammarly in my IDE lol

Reply:
If you're using vim, `set: spell` will catch the majority of spelling mistakes.

Reply:
that's nice, I didn't know that